modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -21,6 +21,11 @@
 		],
 		"to": "getFriendlyURL(new InfoItemReference(param#0#, param#1#), param#2#)"
 	},
+	{
+		"from": "JSONUtil.isValid(String paramName)",
+		"issueKey": "LPD-8007",
+		"to": "JSONUtil.isJSONObject(param#0#)"
+	},
```

Comment:
hi @kyle-miho when the parameter will not be changed and will only be a rename it is not necessary to add this information, and when the class will not be changed it is not added in the “`from`” attribute, it is only used for verification by being placed in “`classNames`”.  e.g.:
```
	{
		"classNames": [
			"JSONUtil"
		],
		"from": "isValid",
		"issueKey": "LPD-8007",
		"to": "isJSONObject"
	},
```
Parameter checking would be relevant, for example, if the class had more than one `isValid` method and only one had been replaced by the other.

Reply:
Another Process that I would Like to suggest is:
The pattern when registered on the ticket is based on the specific Upgrade execution that was found.

Some patterns could happen on other file types and it's up to the judgment of the developers to identify these cases. 
This case, for example, being in a utility class, could appear in JSPs, JSPFs, etc. Of course, we can't guess which cases will come along and we really should worry about what we've already found, but it's interesting at least to look in the Liferay Portal code at the uses of this class and/or method. 
If they are present in other file types, it's worth evaluating the likelihood of them occurring in that other type vs. the cost of developing them.
If:
Cost is high but so is value, it's a good case to analyze.
If the cost is low and the value is high, is valid to develop. And both having a low cost, it also makes sense.

In this scenario we would have: 

```
	{
		"from": "JSONUtil.isValid",
		"issueKey": "LPD-8007",
		"to": "JSONUtil.isJSONObject",
		"validExtensions": [
			"java",
			"jsp"
		]
	},
```